ios/RNGoogleSignin/RNGoogleSignin.m

```diff
@@ -167,7 +167,7 @@ - (void)resolveWithUserDetails: (GIDGoogleUser *) user {
                            @"idToken": user.authentication.idToken,
                            @"serverAuthCode": user.serverAuthCode ? user.serverAuthCode : [NSNull null],
                            @"accessToken": user.authentication.accessToken,
-                           @"scopes": user.accessibleScopes,
+                           @"scopes": user.grantedScopes,
```

Comment:
This is not backwards compatible though, just to let you know. The project won't build on 4.1.2 only on 4.3.0. I'm fine with that but we might need to warn users.

Reply:
I added a note about that in the guide, thanks a lot for pointing this out!